PubnativeLite/PubnativeLite/Ad Request/HyBidAdRequest.m

```diff
@@ -323,9 +327,36 @@ - (void)processResponseWithData:(NSData *)data {
         } else if ([PNLiteResponseOK isEqualToString:response.status]) {
             NSMutableArray *responseAdArray = [[NSArray array] mutableCopy];
             for (HyBidAdModel *adModel in response.ads) {
-                HyBidAd *ad = [[HyBidAd alloc] initWithData:adModel];
+                HyBidAd *ad = [[HyBidAd alloc] initWithData:adModel withZoneID:self.zoneID];
                 [[HyBidAdCache sharedInstance] putAdToCache:ad withZoneID:self.zoneID];
                 [responseAdArray addObject:ad];
+                switch (ad.assetGroupID.integerValue) {
```

Comment:
@faresbh aren't we overriding here the winner behaviour of the IQV SDK between the parallel requests? On Android we decided to not add the change in this class when merging 'cause we didn't have a clear decision on the flow regarding when do we report the win, if before processing the VAST or after. And we'd need to apply the same logic to Adcel.
If we consider the win happens after parsing the VAST it can be tricky 'cause parson could take a long time if there are wrappers. But on the other side if the VAST is faulty we'll still report the win and might loose and opportunity if the other network responds with a good one. That's why we decided to skip that part for now in this class.

Reply:
By skipping it you mean the whole switch case should be removed?

Reply:
I mean not adding the video caching logic in the HyBidAdRequest class for IQV since it overlaps with the win/lose logic of the parallel requests.